### PR TITLE
feat(sdd): reverted changes back to have SDD again

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
We are returning back SDD, beside of this frontend  PR BE guys will turn it on on BE

## Testing Steps (optional)
Create a new wallet; go over SDD flow, make sure SDD flow works as expected

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/index.tsx [hunk 1]
```diff
@@ -30,18 +31,50 @@ class CheckoutConfirm extends PureComponent<Props> {
...
+          const card = cards[0]
+          return this.props.simpleBuyActions.confirmSBCreditCardOrder(
+            card.id,
+            this.props.order
+          )
+        }
+        return this.props.simpleBuyActions.setStep({
+          step: 'ADD_CARD'
+        })
+      } else {
```
Comment (dsisto-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/index.tsx:71]: No need for this `else` as previous lines will `return`

**Review by dsisto-bc — APPROVED** [review]
LGTM!

File: packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts [hunk 2]
```diff
@@ -1069,7 +1069,6 @@ export default ({
         // REMOVE THIS WHEN BACKEND CAN HANDLE PENDING 'FUNDS' ORDERS
         // 👇--------------------------------------------------------
         case 'BANK_ACCOUNT':
-          // case 'USER_CARD': commenting out until we figure out SDD issues
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts:1072]: shouldnt this be added back? it was there for SDD

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts:1072]: @schnogz I'll check since we have BE ready now

**Review by schnogz — APPROVED** [review]

**Pull request merged**